### DISCUSSION TITLE
harvester harvester               : INFO     Failed to connect to PeerInfo(_ip=IPv4Address('127.0.0.1'), _port=8447)

### DISCUSSION THREAD

**Discussion by simonturton** [Chia-Network/chia-blockchain#17824] [Support]
I see this issue reported a few times but cannot find a reliable resolution
at worst it seems i need to rebuild the database 
firewall is off
no conflicting applications
so this is a pure chia application error
please help

**Comment by BrandtH22** [comment]
Hey @simonturton , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

This looks like a configuration or constants issue, what I would do is:
- stop chia
- rename `~/.chia` to `~/.chia_old`
- run chia init (this creates a new .chia folder)
- move your db from `~/.chia_old/mainnet/db/` to `~/.chia/mainnet/db/`
- start chia and let it sync fully

Note this process will create a new config file in the new .chia directory meaning some data will be missing and need to be readded, let us know what information is missing and we can help get it added back 👍

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
